packages/react-components/react-theme/stories/Theme/colors/ThemeColors.stories.tsx

```diff
@@ -98,7 +98,7 @@ const ColorButton: React.FunctionComponent<
 );
 
 const neutralTokens = (Object.keys(theme.webLight) as Array<keyof Theme>).filter(tokenName =>
-  tokenName.match(/^color(?!Palette).*/),
+  tokenName.match(/^colorNeutral(Background|Stroke)Alpha.*/),
```

Comment:
If you want to be more specific but still include all colors that were previously included you should do:

```suggestion
  tokenName.match(/^color(Neutral|Brand|CompoundBrand|Subtle|Transparent)?(Background|Foreground|Scrollbar|Shadow|Stencil|Stroke)(Alpha|Overlay)?.*/),
```

You can check that this works [here](https://regex101.com/r/jNyPbX/1)

Reply:
Actually, this is just a debugging code which should have not been pushed.

Reply:
removed